modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaGetFileMethodCheck.testjava

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetFileMethodCheck {
+
+	public void method() {
+
+    	// Test local service class method
+
+		File file = dlFileEntryLocalService.getFile(fileEntryId, fileEntryVersion, true);
+
+		// Test utility class method
+
+		return DLFileEntryLocalServiceUtil.getFile(fileEntryId, fileEntryVersion, true);
```

Comment:
There is too much indentation here. Is the excessive indentation intentional? In either case, keep the indentation consistent (e.g. the comment does not have the same level of indentation).

Reply:
This is very strange because it's like this:
![Selection_009](https://github.com/kevhlee/liferay-portal/assets/89744445/813428ec-8bdb-4882-8ec6-7a8b042c9b61)

Reply:
What IDE/editor are you using? How your editor visualizes indentation via spaces versus tabs could be different.

Reply:
IntelliJ. Sometimes GitHub shows it differently, but I saw that this branch has a conflict, so I will resubmit. Maybe is different now.

Reply:
For IntelliJ, on the bottom right corner, there is a setting for the indent. I believe that the setting should set to `Tab` with `4` used for "Tab size", "Indent", and "Continuation indent".